Intact.BuinessLogic/Data/Models/RoomDao.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel.DataAnnotations;
+using Intact.BusinessLogic.Data.Enums;
+
+namespace Intact.BusinessLogic.Data.Models;
+
+public class RoomDao
+{
+    [Key]
+    public int Id { get; init; }
+    
+    [StringLength(32)]
+    public string Title { get; init; }
+    
+    public Guid Creator { get; set; }
```

Comment:
is this a user guid ? I think it is better to have Profile reference here, need to think